src/runtime/TemporalObject.h

```diff
@@ -29,6 +29,13 @@
 
 namespace Escargot {
 
+static const int64_t const_hourToNanosecond = 3600000000000;
+static const int64_t const_minuteToNanosecond = 60000000000;
+static const int64_t const_secondToNanosecond = 1000000000;
+static const int64_t const_millisecondToNanosecond = 1000000;
+static const int64_t const_microsecondToNanosecond = 1000;
```

Comment:
Its better to move these variables into `TemporalInstant` class.

About names, how about naming it like `HourToNanosecond`?
We do not prefer using '_' in variable name.